test/integration/slab_sizes.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Some screwy-looking slab sizes.
+export MALLOC_CONF="slab_sizes:1-4096:17|100-200:1|128-128:2"
```

Comment:
What do you think about adding another set of default "aggressive" parameters? i.e. instead of using minimal number of pages, that option will use max or high number of pages, and we can enable it through a single malloc_conf option. It can make the feature easier to use.